ui/src/app/components/breadcrumb.tsx

```diff
@@ -415,12 +422,12 @@ export const Breadcrumb = fp.flow(
 
   return (
     <>
-      {showInvalidBillingBanner && (
-        <InvalidBillingBanner
+      {enableInitialCreditsExpiration && <InvalidBillingBanner />}
```

Comment:
This is where we decide whether to use the new or old banners.

Reply:
A naive read here would make me think "a banner is always shown", but since that is not the case, what do you think about renaming this to something like "InvalidBillingBannerMaybe"?

Alt: a comment here like "shows banner if appropriate" would also work

Reply:
Good call